### DISCUSSION TITLE
Add ability to auto-update

### DISCUSSION THREAD

**Discussion by SwiftWinds** [pdm-project/pdm#602] [Ideas]
It'd be nice if there were some auto-update feature. How `npm` and `pip` does it is by showing the command to update itself at the end of any install command. I think adding this as a feature to PDM would also be a user-friendly way to keep PDM up to date.

**Comment by pawamoy** [comment]
This is was released in [v1.8.1](https://github.com/pdm-project/pdm/releases/tag/1.8.1) :slightly_smiling_face:

**Reply by SwiftWinds** [reply]
Thanks! :)
